.github/workflows/release.yml

```diff
@@ -1,25 +1,28 @@
 name: Release
 on:
   release:
-    types: [published]
+    types: [prereleased]
```

Comment:
:-1: for prereleases (they generate misleading notifications for subscribers AFAIK).

Could we do `draft` instead? https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release implies it's not an option, however as a work-around we can have `release.yml` manually called by the draft-release workflow?

Reply:
> _Could we do `draft` instead?_

We can't: there is no trigger for draft releases.

> _we can have `release.yml` manually called by the draft-release workflow?_

There isn't a “draft-release” workflow either.

Reply:
> _they generate misleading notifications for subscribers_

Indeed, at least for emails, which are immutable:

* *+Before:** [iterative/terraform-provider-iterative] Release v0.11.9 - v0.11.9
* **After:** [iterative/terraform-provider-iterative] Pre-release v0.11.9 - v0.11.9

I wonder if that's more concerning than technical robustness, though.

Reply:
* Moved to https://github.com/iterative/terraform-provider-iterative/pull/677